nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/GetNotificationConfiguration.java

```diff
@@ -91,32 +88,46 @@ public void execute(AdminCommandContext context) {
 
         String headers[] = {"Enabled", "Notifier Enabled", "Notifier Name"};
         ColumnFormatter columnFormatter = new ColumnFormatter(headers);
-        Object values[] = new Object[3];
-        values[0] = notificationServiceConfiguration.getEnabled();
 
-        for (ServiceHandle<BaseNotifierService> serviceHandle : allServiceHandles) {
-
-            NotifierConfiguration notifierConfiguration = configuration.getNotifierConfigurationByType(serviceHandle.getService().getNotifierConfigType());
-            LogNotifierConfiguration logNotifierConfiguration = (LogNotifierConfiguration) notifierConfiguration;
-            if (logNotifierConfiguration == null) {
-                values[1] = "true";
-            } else {
-                values[1] = logNotifierConfiguration.getEnabled();
+        if (configuration.getNotifierConfigurationList().isEmpty()) {
+            mainActionReport.setMessage("No notifier defined");
+        }
+        else {
+            List<Class<NotifierConfiguration>> notifierConfigurationClassList = Lists.transform(configuration.getNotifierConfigurationList(), new Function<NotifierConfiguration, Class<NotifierConfiguration>>() {
+                @Override
+                public Class<NotifierConfiguration> apply(NotifierConfiguration input) {
+                    return resolveNotifierConfigurationClass(input);
+                }
+            });
+
+            Properties extraProps = new Properties();
+            for (ServiceHandle<BaseNotifierService> serviceHandle : allServiceHandles) {
+                NotifierConfiguration notifierConfiguration = configuration.getNotifierConfigurationByType(serviceHandle.getService().getNotifierConfigType());
+                if (notifierConfigurationClassList.contains(resolveNotifierConfigurationClass(notifierConfiguration))) {
+                    Object values[] = new Object[3];
+                    values[0] = notificationServiceConfiguration.getEnabled();
+                    values[1] = notifierConfiguration.getEnabled();
+                    String serviceName = serviceHandle.getActiveDescriptor().getName();
+                    values[2] = serviceName;
+                    columnFormatter.addRow(values);
+
+                    Map<String, Object> map = new HashMap<>(3);
+                    map.put("enabled", values[0]);
+                    map.put("notifierEnabled", values[1]);
+                    map.put("notifierName", values[2]);
+
+                    extraProps.put("getNotificationConfiguration-" + serviceName, map);
```

Comment:
If you're testing the admin console btw, you'll need the fixes introduced in https://github.com/payara/Payara/pull/1304